exercises/sending-signals-external/solution/workflow.py

```diff
@@ -0,0 +1,86 @@
+import asyncio
+from datetime import timedelta
+
+from temporalio import workflow
+from temporalio.exceptions import ApplicationError
+
+# Import activity, passing it through the sandbox without reloading the module
+with workflow.unsafe.imports_passed_through():
+    from activities import PizzaOrderActivities
+    from shared import Bill, OrderConfirmation, PizzaOrder
+
+@workflow.defn
+class PizzaOrderWorkflow:
+    def __init__(self) -> None:
+        self._pending_confirmation: asyncio.Queue[str] = asyncio.Queue()
+
+    @workflow.run
+    async def order_pizza(self, order: PizzaOrder) -> OrderConfirmation:
+        workflow.logger.info(f"order_pizza workflow invoked with {input}")
```

Comment:
where is input defined?

Reply:
See above; fixed in https://github.com/temporalio/edu-interacting-with-workflows-python-code/pull/1/commits/1e4e511920338efea3ce234b4a948fc8e78ff82b